### PR TITLE
LPS-160386 No option to bulk deactivate or activate countries and regions

### PR DESCRIPTION
**Pull request by RafaelCRC**
Update for [LPS-160386](https://issues.liferay.com/browse/LPS-160386)

### CODE REVIEW

**Review by gislayne-vitorino — APPROVED** [review]
LGTM!

File: modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/UpdateCountryStatusMVCActionCommand.java [hunk 1]
```diff
@@ -37,24 +41,37 @@
...
+	}
 
 	@Override
 	protected void doProcessAction(
 			ActionRequest actionRequest, ActionResponse actionResponse)
 		throws Exception {
 
-		long countryId = ParamUtil.getLong(actionRequest, "countryId");
+		long[] countryIds = StringUtil.split(
+			ParamUtil.getString(actionRequest, "countryIds"), 0L);
```
Comment (drewbrokke) [inline modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/UpdateCountryStatusMVCActionCommand.java:64]: We should prefer using `ParamUtil.getLongValues` here.

File: modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/UpdateRegionStatusMVCActionCommand.java [hunk 2]
```diff
@@ -37,24 +41,37 @@
...
+	}
 
 	@Override
 	protected void doProcessAction(
 			ActionRequest actionRequest, ActionResponse actionResponse)
 		throws Exception {
 
-		long regionId = ParamUtil.getLong(actionRequest, "regionId");
+		long[] regionIds = StringUtil.split(
+			ParamUtil.getString(actionRequest, "regionIds"), 0L);
```
Comment (drewbrokke) [inline modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/UpdateRegionStatusMVCActionCommand.java:64]: Same here, use `ParamUtil.getLongValues`.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@RafaelCRC looks great, just a couple small changes.

File: modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/UpdateRegionStatusMVCActionCommand.java (hunk 2 above)
Comment (RafaelCRC) [reply modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/UpdateRegionStatusMVCActionCommand.java]: i updated the code @drewbrokke, thanks!

**Review by drewbrokke — APPROVED** [review]
LGTM! 👍

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**